src/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -68,15 +76,39 @@ export const mockEntries: mockData[]= [
 
 const DashboardPage = () => {
     const user = useContext(AuthContext);
+    const navigate = useNavigate();
+    const foundEntries = 
+        mockEntries.filter((entry: mockData) => {
+            return user.id === entry.userId
+        }
+    )
+
```

Comment:
This is where the call to the backend will be, it will return an array of entries where the user.id matches the entry.userId aka the creator of the entry